frontend/public/components/edit-yaml.jsx

```diff
@@ -278,47 +305,69 @@ export const EditYAML_ = connect(stateToProps)(
           });
         }
 
-        save() {
-          const { onSave, t } = this.props;
-          const { owner } = this.state;
-          let obj;
+        performDryRun(objs) {
+          const { models } = this.props;
+          let dryRunResources = [];
```

Comment:
This name is a little misleading since they're not resources, they're promises. Since it's only used in `setState`, can we initialize it as a `const` there?

```ts
const promises = createResources(objs, models, true);
```

Reply:
Or you could call them `requests` if that's more clear

Reply:
I think the code block changed, const makes more sense now, updated the name too